### ISSUE TITLE
Add indicator (visual and non-visual) of link leading to another site

### ISSUE THREAD

**Issue by eevajonnapanula** [turkupy/turkupy.github.io#62]
**Is your feature request related to a problem? Please describe.**
There is no visual (or non visual) indicator of a link leading to another site, so that is needed. 

**Describe the solution you'd like**
A separate component (called, for example, `ExternalLink`) which wraps the link, and contains a visual indicator (for example https://fontawesome.com/icons/external-link-alt) to communicate about the link to external site, and also a no-visual indicator (such as visually hidden text) telling the same thing. Also note, that this text needs to be translatable. 

A note here: As we are not currently using any other Font Awesome-icons, maybe a SVG-file of the icon used is enough.

**Additional context**
Why this should be? [WebAIM provides some more context](https://fontawesome.com/icons/external-link-alt)

**Comment by robertatiriticco** [comment]
Can I be assigned to this ticket?

**Comment by eevajonnapanula** [comment]
Hey @robertatiriticco, how it is going with the issue? Is there anything we could do to help you with it? 😊
